test/models/Stdlib.test.js

```diff
@@ -50,8 +52,8 @@ contract('Stdlib', function () {
       this.stdlib.name.should.eq(packageName)
     })
 
-    it('should set a version', async function () {
-      this.stdlib.version.should.eq('1.1.0')
+    it('should not set a version if not specified', async function () {
+      should.not.exist(this.stdlib.version)
```

Comment:
I'm not sure about this scenario. It's kind of weird installing a dependency and not seeing which version I got installed in my package-file. Maybe because I'm used to those package managers that assume you requested the latest version if you didn't specify one. For example, NPM will assume you're requesting the latest including minor and patch updates. Do you agree?

Reply:
Sounds good!